USAGE.md

```diff
@@ -0,0 +1,65 @@
+# SyntaxTree Usage
+
+## Utilizing SyntaxTree's API
+
+Aside from providing a formatted view of AST nodes, SyntaxTree also provides access to information on each AST node.
+
+This quick tutorial will show you some common APIs in SyntaxTree. Keep in mind this is only a fraction of methods available to use and you can find more in the [official documentation](https://ruby-syntax-tree.github.io/syntax_tree/).
+
+Alright! Let's attain the `:+` operator in the following Ruby source code:
+
+```ruby
+require "syntax_tree"
+tree = SyntaxTree.parse("puts 1+1")
```

Comment:
Maybe we can use the name of the node we get here:

```suggestion
program = SyntaxTree.parse("puts 1+1")
```

This will help in the following example to explain where `statements` is coming from.

Reply:
Maybe we could also do a `pp program` and show the output in this doc so readers can follow along which nodes we're accessing? Or just show the output like you did in other code snippets:

```ruby
# => (program (statements (command (ident "puts") (args ((binary (int "1") :+ (int "1")))))))
```